DSharpPlus.HttpInteraction.AspNetCore/DSharpPlus.HttpInteraction.AspNetCore.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
```

Comment:
missing `<IsPackable>true</IsPackable>`, shouldn't explicitly specify the target framework